src/Symfony/Component/EventDispatcher/DependencyInjection/AddEventAliasesPass.php

```diff
@@ -26,6 +26,10 @@ class AddEventAliasesPass implements CompilerPassInterface
 
     public function __construct(array $eventAliases, string $eventAliasesParameter = 'event_dispatcher.event_aliases')
     {
+        if (1 < \func_num_args()) {
+            trigger_deprecation('symfony/event-dispatcher', '5.3', 'Configuring "%s" is deprecated.', __CLASS__);
```

Comment:
It feels like it still can be configured with `$eventAliases`, what do you think about mentioning `$eventAliasesParameter` in the deprecation message?